[Bug]: jupyter book init --gh-pages

### What happened, and what did you expect to happen?

I used `jupyter book init --gh-pages` to generate a [workflow](https://github.com/ds-modules/modules-textbook/commit/3b97e543c2a262612a4facbdf8db371e514ad9dc) for a GitHub Pages site, but this workflow [doesn't run as expected](https://github.com/ds-modules/modules-textbook/actions/runs/16280597549). I found this command in [the docs](https://next.jupyterbook.org/start/publish). It seems to be a derivative of the `myst init --gh-pages` command. 

The workflow that ultimately worked for me: https://github.com/ds-modules/modules-textbook/commit/0a5ee99a6fcabbd6efb2f385a7ace36bda0be01f
With output [here](https://github.com/ds-modules/modules-textbook/actions/runs/16280806954/job/45969855601)

### What version of Jupyter Book are you running?

v2.0.0-b1

### What Operating System are you using?

macOS

### Relevant log output

```shell
Run jupyter book build --html
/home/runner/work/_temp/e9c416b3-c7e9-4691-9409-9caa985eb4d0.sh: line 1: jupyter: command not found
Error: Process completed with exit code 127.
```

--- a/ts/index.ts
+++ b/ts/index.ts
@@ -26,8 +26,10 @@ const program = new Command();
 // Set whitelabelling
 process.env.MYSTMD_READABLE_NAME = "Jupyter Book";
 process.env.MYSTMD_BINARY_NAME = "jupyter book";
-process.env.MYSTMD_NPM_PACKAGE_NAME = "jupyter-book";
 process.env.MYSTMD_HOME_URL = "https://next.jupyterbook.org";
+// Specfically for NPM
+process.env.MYSTMD_NPM_BINARY_NAME = "jupyter-book";
+process.env.MYSTMD_NPM_PACKAGE_NAME = "jupyter-book";
 
 program.description(
   `Jupyter Book is powered by ${chalk.blue(

